File: initializr-generator/src/main/resources/templates/ServletInitializer.java
package {{packageName}};

import org.springframework.boot.builder.SpringApplicationBuilder;
{{servletInitializrImport}}

public class ServletInitializer extends SpringBootServletInitializer {

	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources({{applicationName}}.class);
	}

}

File: initializr-generator/src/main/resources/templates/ServletInitializer.kt
package {{packageName}}

import org.springframework.boot.builder.SpringApplicationBuilder
{{servletInitializrImport}}

class ServletInitializer : SpringBootServletInitializer() {

	override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
		return application.sources({{applicationName}}::class.java)
	}

}

